### PR TITLE
WebGL: Fix MSAA depth/stencil textures not working

### PR DESCRIPTION
**Pull request by Popov72**
This PG does not currently work: https://playground.babylonjs.com/#BP3T4T#11

```md
[.WebGL-000019CC00653F00] GL_INVALID_FRAMEBUFFER_OPERATION: Framebuffer is incomplete: Attachments have different sample counts.
```

This PR fixes it, and also allows to create MSAA `RenderTargetWrapper` with only the depth/stencil texture and no color texture.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15508/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15508/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15508/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15508/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15508/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15508/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15508/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15508/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15508/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15508/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts [hunk 1]
```diff
@@ -109,7 +102,7 @@ ThinEngine.prototype.createRenderTargetTexture = function (this: ThinEngine, siz
     return rtWrapper;
 };
 
-ThinEngine.prototype._createDepthStencilTexture = function (size: TextureSize, options: DepthTextureCreationOptions): InternalTexture {
+ThinEngine.prototype._createDepthStencilTexture = function (size: TextureSize, options: DepthTextureCreationOptions, rtWrapper: WebGLRenderTargetWrapper): InternalTexture {
```
Comment (sebavan) [inline packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts:105]: How does this impact native ?

Comment (Popov72) [reply packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts:105]: It does not impact Native, as this parameter was already in the signature of the method, but not used in WebGL / WebGPU (but used in Native!).

**Pull request merged**